### ISSUE TITLE
Users - scroll UI

### ISSUE THREAD

**Issue by nechalmaggon** [ToolJet/ToolJet#9956]
**Where?**
- User list container 

**What to fix?** 
- Dark mode 
- Scroll to be visible and shown only on hover of the container and not constantly (for Chrome and Safari browsers)
- Text should not be visible in the space above the table headers. Refer to the screenshot below.

![Image](https://github.com/user-attachments/assets/bd6d28f9-2e01-441f-82cc-9dca1a0e6594)

**Comment by Ramzi-Abidi** [comment]
Hi @nechalmaggon,

Can I take this issue ?

**Comment by nechalmaggon** [comment]
Hi @Ramzi-Abidi thank you for showing interest, however, this issue will ber resolved internally. Do look out for issues with the label `up-for-grabs` - those are the ones open to the community!

**Comment by CuriousCoder00** [comment]
@PriteshKiri There's also this one issue where text is visible behind the header after scrolling
![image](https://github.com/user-attachments/assets/45984b50-034a-4a8e-aac6-0998cac46324)

**Comment by CuriousCoder00** [comment]
I've found the cause and resolved it.. would like to work on the remaining problems..
I would like to work on it..

**Comment by CuriousCoder00** [comment]
> **Where?**
> 
> * User list container
> 
> **What to fix?**
> 
> * Dark mode
> * Scroll to be shown only on hover of the container and not constantly
> * Page to be fixed and should not move up & down on scroll attempt- only container should scroll
> 
> <img alt="Screenshot 2024-06-03 at 3 22 16 PM" width="1440" src="https://private-user-images.githubusercontent.com/127945292/336024643-288b1936-3682-43ae-ac4d-5e9f74231f4a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE5MjU3NDksIm5iZiI6MTcyMTkyNTQ0OSwicGF0aCI6Ii8xMjc5NDUyOTIvMzM2MDI0NjQzLTI4OGIxOTM2LTM2ODItNDNhZS1hYzRkLTVlOWY3NDIzMWY0YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyNVQxNjM3MjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01NjNjYjU1MDcxYTQ3NTgzNDFmYzNkMGIxMGM3N2NmNzMwYWUwOWI2ZDNmNDdlMDQ4OGM2YjFlMDMyMjJiYjRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.5Fm_sFDNDsLD8CxLpbpldl6RieBOSn9oci-jxoj_ReM">
>  Screen.Recording.2024-06-03.at.3.32.52.PM.mov

These issue are in Version `2.38.0-e2.17.0`
Not sure about safari though...!!!

**Comment by CuriousCoder00** [comment]
Cuzz there's no issue in latest version

**Comment by PriteshKiri** [comment]
@CuriousCoder00 the scroll bars are not visible in Chrome and Safari. Just updated the issue. Also, please let me know if you are interested in working on this issue.  I will assign it to you.

It would be nice if you could kindly wait for the issue to be assigned before starting on the PR, it would help us ensure everything is properly coordinated.

**Comment by CuriousCoder00** [comment]
I understand.. I would like to work on this issue.
For the issue:
`Text should not be visible in the space above the table headers. Refer to the screenshot below.`
there is a `padding-top:6px;` css added so that's causing this issue.

and for the scroll bar - should i make it visible, when i hover the container only?
I'll have to look into it a bit..

@PriteshKiri can you assign it to me?

**Comment by PriteshKiri** [comment]
Assigned @CuriousCoder00. Happy coding :)

Yes, scroll bars should only be visible when hovering on the containers.

**Comment by CuriousCoder00** [comment]
@PriteshKiri should i do the same for light mode? Or it should only be implemented with dark mode??

**Comment by PriteshKiri** [comment]
@CuriousCoder00 it should be for both dark and light modes.

**Comment by CuriousCoder00** [comment]
I can see here that the `-webkit-scrollbar` is set to `display-none`

**Comment by CuriousCoder00** [comment]
If there's no scrollbar when it's not hovering then it might cause layout shift..

**Comment by CuriousCoder00** [comment]
@PriteshKiri one last query..!!
Shall it only be implemented to current page.. `workspace settings > users` ?

**Comment by PriteshKiri** [comment]
You can try doing that and create a PR of the same. Also, share the video in your PR about the same. If it is causing any issues we can have the discussion in the PR discussion.

>workspace settings > users.  
Yes, this is correct

**Comment by CuriousCoder00** [comment]
@PriteshKiri sure.. will do a pr in evening

**Comment by CuriousCoder00** [comment]
I've made the required changes... It's working..
but it's showing my recent commits as well that i made in previous PR  #10467 
Is that gonna be an issue? @PriteshKiri

**Comment by PriteshKiri** [comment]
@CuriousCoder00 I can see changes in both issues in this PR. Can you please create new PR or cherry-pick changes in PR for this issue?

**Comment by CuriousCoder00** [comment]
Will make a new PR or try to cherry-pick changes

**Comment by CuriousCoder00** [comment]
It's causing an issue as i by fault created a branch from tooltip issue... and i can't seem to cherry pick the commits
tbh i'm new to open source contributions.. any solution or directions??

**Comment by PriteshKiri** [comment]
Hey @CuriousCoder00, you can create a new branch from the base branch and can make the changes fixing this issue. And then create a new PR for the same. Just see the file changes and do it again on the new branch you created from the main.

**Comment by CuriousCoder00** [comment]
@PriteshKiri Hey! wanted to update on the status of this issue i tried multiple things, seems like i messed up with the main branch itself.. 
It's showing all the commits 
and changes in two files....
![image](https://github.com/user-attachments/assets/d31d2c97-9d5d-45a4-aa16-6c20f8d5b97d)
![image](https://github.com/user-attachments/assets/e1f4ee5b-71aa-42d9-8ab3-f126b7c3d1e9)

**Comment by CuriousCoder00** [comment]
I'mma create a new fork and then work again on all the issues.. would that work??? Please ping me if that's fine.. i'll pull PR after making a new fork

**Comment by CuriousCoder00** [comment]
@PriteshKiri I've made required changes and created a PR for this issue.
Do let me know if there are any required changes.

**Comment by CuriousCoder00** [comment]
@PriteshKiri any update on this issue??

**Comment by PriteshKiri** [comment]
Hey @CuriousCoder00  ,

This might take some time, but I’ll definitely inform you once it's reviewed and merged. Thanks for your patience!

**Comment by CuriousCoder00** [comment]
@PriteshKiri It's been about a month, any update?

**Comment by PriteshKiri** [comment]
Hey @CuriousCoder00 ,

Apologies for the delay. The PR should be merged anytime within this month. 

Thanks for your patience!
